.gitignore

```diff
@@ -5,3 +5,4 @@ node_modules
 .hoodie
 .nyc_output
 docs/_build/
+.vscode
```

Comment:
can you leave this out of the `.gitignore`? You can add it to a `.gitignore` in folder above the repository. I have things like `node_modules` in `~/.gitignore`